KarhooUISDK/Screens/BookingRequestScreen/FormBookingRequestScreen/PassengerDetails/PassengerDetailsView.swift

```diff
@@ -117,6 +121,14 @@ final class PassengerDetailsView: UIView {
                                    trailing: -textFieldInset)
         }
     }
+    
+    private func textFieldValuesChanged()
+    {
+        if let phoneNumberAccessibilityIdentifier = phoneTextField.accessibilityIdentifier {
```

Comment:
What if the phone number is provided but not the email? Would this still work?

Reply:
Yes, the validation would still be triggered. It doesn't matter what text field triggers input validation because the existing validation method checks all the mandatory fields every time before deciding to enable / disable the Book button. So if no email is provided, the button will not be enabled until that information is added (intended behavior). I chose the phone number as a precaution because it's the last of the mandatory fields. Other fields may gain more functionality over time, like opening the keyboard for the next field on the return event, but this behavior should not apply to the phone number since it's the last mandatory one.